Makefile

```diff
@@ -66,3 +70,19 @@ publish:
 	@docker login
 	@docker push $(IMAGE_REPO):$(TAG)
 	@docker push $(IMAGE_REPO):latest
+
+# Install KIND
+install-kind: system-check
+	@chmod +x hack/kind-cluster.sh
+	@./hack/kind-cluster.sh install-kind
+
+# Create KIND cluster
+create-kind: system-check
+	@chmod +x hack/kind-cluster.sh
```

Comment:
We don't need to chmod each time. You can commit the file with executable permissions

Reply:
Done